opba-facade-protocol-api-shared/src/main/java/de/adorsys/opba/protocol/api/dto/KeyWithParamsDto.java

```diff
@@ -0,0 +1,27 @@
+package de.adorsys.opba.protocol.api.dto;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.ToString;
+
+@Getter
+@EqualsAndHashCode(callSuper = true)
```

Comment:
Don't see why we would check key equality using Dto

Reply:
removed